user-interface/src/main/resources/impressum/PrivacyAgreement.html

```diff
@@ -0,0 +1,834 @@
+    based on our legitimate interests in an efficient and secure mailing system.</p>
+  <strong>Contents:</strong>
+  <p>Information about planned maintenance or new features.</p>
+  <ul class="m-elements">
+    <li><strong>Processed data types:</strong> Inventory data (For example, the full name,
+      residential address, contact information, customer number, etc.); Contact data (e.g. postal
+      and email addresses or phone numbers). Meta, communication and process data (e.g. IP
+      addresses, timestamps, identification numbers, involved parties).
+    </li>
+    <li><strong>Data subjects:</strong> Communication partner (Recipients of e-mails, letters,
+      etc.).
+    </li>
+    <li><strong>Purposes of processing:</strong> Direct marketing (e.g. by e-mail or postal).</li>
+    <li><strong>Retention and deletion:</strong> 3 years - Contractual claims (AT) (Data required to
+      consider potential warranty and compensation claims or similar contractual claims and rights,
+      as well as to process related inquiries, based on previous business experiences and common
+      industry practices, will be stored for the duration of the regular statutory limitation period
+      of three years (Sections 1478, 1480 of the Austrian Civil Code).). 10 years - Contractual
+      claims (CH) (Data required to consider potential compensation claims or similar contractual
+      claims and rights, as well as to process related inquiries, based on previous business
+      experiences and common industry practices, will be stored for the duration of the statutory
+      limitation period of ten years, unless a shorter period of 5 years is applicable, which is
+      relevant in certain cases. This period begins at the end of the calendar year in which the
+      claim arose (Articles 127 and 128 Swiss Code of Obligations (CO))).
+    </li>
+    <li class=""><strong>Legal Basis:</strong> Consent (Article 6 (1) (a) GDPR).</li>
+    <li><strong>Opt-Out: </strong>You can cancel the receipt of our newsletter at any time, i.e.
+      revoke your consent or object to further receipt. You will find a link to cancel the
+      newsletter either at the end of each newsletter or you can otherwise use one of the contact
+      options listed above, preferably e-mail.
+    </li>
+  </ul>
+  <h2 id="m15">Changes and Updates</h2>
+  <p>We kindly ask you to inform yourself regularly about the contents of our data protection
+    declaration. We will adjust the privacy policy as changes in our data processing practices make
+    this necessary. We will inform you as soon as the changes require your cooperation (e.g.
+    consent) or other individual notification.</p>
+  <p>If we provide addresses and contact information of companies and organizations in this privacy
+    policy, we ask you to note that addresses may change over time and to verify the information
+    before contacting us.</p>
+
+  <h2 id="m42">Terminology and Definitions</h2>
```

Comment:
Please make sure this is in sync with the data types mentioned in the introduction and contains a definition for all terms

Reply:
done